src/lib/components/BottomDrawer.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
```

Comment:
Made this component before realizing the UI had changed to be fullscreen modals. Going to include it since we'll probably want to refactor existing CupertinoPane usages over to it.